### PR TITLE
Lit: Initial lit2 support

### PR DESCRIPTION
**Pull request by brion-fuller**
Issue: #14455 

## What I did

Initial start to setting up lit framework

### CODE REVIEW

**Comment by shilman** [comment]
@gaetanmaisse these last set of changes (addon-docs/babel-loader stuff) look good, but probably should be in their own PR!

**Comment by gaetanmaisse** [comment]
Yep @shilman, I will open a PR with these and maybe some other related stuff ;)

File: examples/lit-kitchen-sink/README.md [hunk 1]
```diff
@@ -3,3 +3,5 @@
 This example directory represents the application you wish to integrate storybook into.
 
 Run `yarn install` to sync Storybook module with the source code and run `yarn storybook` to start the Storybook.
+
+When developing things in `@storybook/lit` you can run `yarn build lit --watch` in the root folder of the monorepo and both changes in the example code or in `@storybook/lit` will be reflected in the UI.
```
Comment (shilman) [inline examples/lit-kitchen-sink/README.md:7]: NICE 💯

**Comment by brion-fuller** [comment]
@gaetanmaisse Looks good to me.

**Review by shilman — APPROVED** [review]
Looking great @brion-fuller @gaetanmaisse 💯

**Pull request merged**